.circleci/config.yml

```diff
@@ -47,6 +40,12 @@ jobs:
                      --password=$SANDBOX_PASSWORD \
                      --client-id=50 \
                      tests
+      - run:
+          name: 'pylint tests'
+          command: |
+            source /usr/local/share/virtualenvs/tap-tester/bin/activate
+            pip install pylint
```

Comment:
So the other pylint step uses tap-facebook's virtual env, but this one uses tap-tester. I don't think it should matter much either way but I think we should be consistent. If you use the tap-facebook one since you already pip installed .[dev] you don't need to install pylint again.

Reply:
I use tap-tester rather than the tap when pyilnting the tests in order to avoid getting `import-errors` in the tests for base.py and test_client.py (if applicable).